db/20190411104800_create_users.rb

```diff
@@ -0,0 +1,8 @@
+class CreateUsers < ActiveRecord::Migration[5:2]
```

Comment:
Are you sure you follow the rails tutorial?

Reply:
Hi I followed steps from chatwork because the steps from the video isn't working.
From chatworks: 
Please fix our comments, and push to github again
when the trainers comment: "merged" -> you can merge that pull request. 
if your pull request merged, you need checkout to main:
git checkout main
git pull origin main

# and create new branch to develop chapter 3
git checkout -b chapter_3
and continue code chapter 3
when you finish coding chapter 3, you push to github same as did with init project
git add .
git commit -m "chapter 3"
git push origin chapter_3

Reply:
Hi I just updated the code in sublime. Is it correct now? attached with a screenshot: 

![Screenshot from 2021-08-23 15-19-30](https://user-images.githubusercontent.com/87750131/130406586-0df08f79-531b-47be-81bd-00d52717eea4.png)

I cannot do exactly like the video with the commands on the video. I'm not sure is it correct or not.